cypress/e2e/features/03_post_user.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Create a Todo Item
+
+  Scenario: Create a new todo item
+    Given I make a POST request to url
```

Comment:
there is already a step implemented for this kind of url navigations, pls use it